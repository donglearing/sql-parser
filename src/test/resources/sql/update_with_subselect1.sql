update T set a = (select f.a from F f)