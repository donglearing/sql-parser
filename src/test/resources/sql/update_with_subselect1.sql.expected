UPDATE T SET a = (SELECT f.a FROM F AS f)